spring-cloud-stream-binders/spring-cloud-stream-binder-kafka/pom.xml

```diff
@@ -63,12 +72,6 @@
 			<groupId>org.springframework.xd</groupId>
 			<artifactId>spring-xd-tuple</artifactId>
 			<version>${spring-xd.version}</version>
-			<exclusions>
```

Comment:
Not sure I get this (haven't looked in detail), but if we're now relying on SI for codec, doesn't that exclusion make *even* more sense now?